libs/remix-ui/file-decorators/src/lib/components/file-decoration-icon.tsx

```diff
@@ -8,44 +8,42 @@ import FileDecorationTooltip from './filedecorationicons/file-decoration-tooltip
 import FileDecorationWarningIcon from './filedecorationicons/file-decoration-warning-icon'
 
 export type fileDecorationProps = {
-    file: FileType,
-    fileDecorations: fileDecoration[]
+  file: FileType,
+  fileDecorations: fileDecoration[]
 }
 
 export const FileDecorationIcons = (props: fileDecorationProps) => {
-    const [states, setStates] = useState<fileDecoration[]>([])
-    useEffect(() => {
-        //console.log(props.file)
-        //console.log(props.fileState)
-        setStates(props.fileDecorations.filter((fileDecoration) => fileDecoration.path === props.file.path || `${fileDecoration.workspace.name}/${fileDecoration.path}` === props.file.path))
-    }, [props.fileDecorations])
+  const [states, setStates] = useState<fileDecoration[]>([])
+  useEffect(() => {
+    //console.log(props.file)
+    //console.log(props.fileState)
```

Comment:
Unused comments

Reply:
i know it was there before i though it maybe needed